### ISSUE TITLE
Crashes latest version of Tales of Arise

### ISSUE THREAD

**Issue by Lulech23** [KingKrouch/AriseFix#1]
After the latest update to Tales of Arise, running AriseFix will display a message that the game has crashed and must close. Oddly enough, the game doesn't actually crash or close until you acknowledge the message. However, continuing to let the game boot results in a black screen for all 3D elements.

Most likely this is an issue of memory offsets changing in the new version. Based on my own digging, it appears the in-game FPS limit is now at address `1B1849A8724` and t.MaxFPS is at `1B1E9035380`. However, modifying these values in realtime had no effect, so perhaps I have the wrong values. 

I have not scanned for the other values, since my only usage of AriseFix was the FPS limiter.

**Comment by Naitrate** [comment]
Yeah, I suspected that it might have been out of date memory pointer offsets for those addresses. I'm planning on addressing this in a way that should work with future patches (by using signature scanning and trampolines).

Once I finish with a stable build of dbzk_fps using this method, I'll start working on updating this mod.

**Comment by Naitrate** [comment]
While looking around, I have determined that the crashing (on my end, although I'm using a not-yet released build) is on the end of ASI Loader (at least in Windows 11). Try removing the plugins folder, and see if the game runs. If it crashes, then that's likely the issue.

I may end up having to find a way to just inject into dinput8 myself, since I've been needing to get around to making a swiss army knife for UE4 games, as I'm tired of updating a million different code repositories.

**Comment by TheTrain** [comment]
As you said, game start whenever you delete .asi file (Without ultrawide fix clearly)
Now bamco also released a new patch so maybe they changed something else too but it is what it is

**Comment by LoneStrider18** [comment]
So has anyone solved this issue to get this mod to work?
